app/src/main/java/edu/uc/forbesne/shoppingsidekick/ui/main/StoreFragment.kt

```diff
@@ -6,32 +6,44 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
```

Comment:
Things I did:
1) I deleted StoreFragemnt
2) Refactored the name of StoreFragment2 to StoreFragment

That is it.